pkg/apis/storageos/v1alpha1/job_types.go

```diff
@@ -35,6 +35,10 @@ type JobSpec struct {
 	// NodeSelectorTerms is the set of placement of the job pods using node
 	// affinity requiredDuringSchedulingIgnoredDuringExecution.
 	NodeSelectorTerms []corev1.NodeSelectorTerm `json:"nodeSelectorTerms"`
+
+	// Tolerations is to set the placement of storageos pods using
```

Comment:
I would like to receive comments if the job needs to be toleration aware too? I did not find the same content in the `job` CRD as within `storageoscluster`.

Reply:
You should also add tolerations in the job controller, similar to this https://github.com/storageos/cluster-operator/blob/v0.0.8/pkg/controller/job/job_controller.go#L283-L290.